controllers/componentversion_controller.go

```diff
@@ -112,6 +111,7 @@ func (r *ComponentVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			obj.Status.ObservedGeneration = obj.Generation
 			msg := fmt.Sprintf("Reconciliation finished, next run in %s", obj.GetRequeueAfter())
 			vid := fmt.Sprintf("%s:%s", obj.Status.ComponentDescriptor.Name, obj.Status.ReconciledVersion)
+			conditions.MarkTrue(obj, meta.ReadyCondition, meta.SucceededReason, "Reconciliation success")
```

Comment:
if we get in that `if` block, it means that `meta.ReadyCondition` is already `true`, that's what `conditions.IsReady(obj)`checks. So why set it here?

Reply:
Agreed, agreed. :D It's weird. I think we need to rethink it.